digital-workspace/2.2/install/index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Installing Digital Workspace
+---
+There are a number of different ways to deploy Alfresco Digital Workspace. You can deploy it using Docker images that are packaged in Helm charts or using Docker Compose. You can also install Alfresco Content Services using standard WAR files contained in the distribution zip, and then configure the installation to include Alfresco Digital Workspace.
+
+The deployment methods are:
+
+* [Containerized deployment]({% link digital-workspace/latest/install/containerized.md %}). Due to the limited capabilities of Docker Compose, this deployment method is recommended for development and test environments only.
+* [Installing into Tomcat]({% link digital-workspace/latest/install/tomcat.md %})
+* [Installing into a different web server]({% link digital-workspace/latest/install/other-webserver.md %})
```

Comment:
In all the ADW 2.2 files, the internal links go to `/latest/` - multiple files need fixing.

Reply:
fixed